app/helpers/orchid/application_helper.rb

```diff
@@ -162,6 +162,55 @@ def prefix_path(path, *args, **kwargs)
     end
   end
 
+  # Render section overrides and localized partials
+  # Partial name does not include the locale, underscore, or extensions
+  # Example:
+  #   render_overridable("explore/partials", "index")
+  #   Looks for in order:
+  #     If @section defined & locale == "es", "@section/_index_es.html.erb"
```

Comment:
thanks for this style of documentation, it is very friendly for how my brain works!